tests/zenko_e2e/backbeat/BackbeatAPIUtility.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const { makeGETRequest, makePOSTRequest, getResponseBody } =
+    require('./utils/request');
+
+class BackbeatAPIUtility {
+    constructor() {}
```

Comment:
If I'm not mistaken, we can omit the constructor if it's not used.

Reply:
Will remove this and test it x)